<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>651</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 220);
color: rgb(218, 234, 243);</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="icon">
     <iconset>
      <normaloff>../../Downloads/th.jpg</normaloff>../../Downloads/th.jpg</iconset>
    </attribute>
    <attribute name="title">
     <string/>
    </attribute>
    <widget class="QLabel" name="username">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>951</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
background-color: rgb(39, 39, 39);
color: rgb(255, 255, 255);
font: 28pt &quot;Roman&quot;;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>your task</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QListWidget" name="TaskListWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>291</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(129, 129, 129);</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
    <widget class="QListWidget" name="TaskListWidget_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>170</y>
       <width>291</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(129, 129, 129);</string>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
    <widget class="QListWidget" name="TaskListWidget_3">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>170</y>
       <width>291</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(129, 129, 129);</string>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>921</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
font: 11pt &quot;Segoe UI&quot;;
background-color: rgb(39, 39, 39);
color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string>To Do</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
font: 11pt &quot;Segoe UI&quot;;
background-color: rgb(39, 39, 39);
color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string>In porgress</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
font: 11pt &quot;Segoe UI&quot;;
background-color: rgb(39, 39, 39);
color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string>Done</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="savechangies">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>480</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(39, 39, 39);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Save Changies</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="icon">
     <iconset>
      <normaloff>../../Downloads/th (2).jpg</normaloff>../../Downloads/th (2).jpg</iconset>
    </attribute>
    <attribute name="title">
     <string/>
    </attribute>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>961</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(129, 129, 129);</string>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="layoutMode">
      <enum>QListView::SinglePass</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>60</y>
       <width>194</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(178, 183, 255);</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>20</hour>
       <minute>30</minute>
       <second>0</second>
       <year>2021</year>
       <month>12</month>
       <day>28</day>
      </datetime>
     </property>
     <property name="time">
      <time>
       <hour>20</hour>
       <minute>30</minute>
       <second>0</second>
      </time>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>2</hour>
       <minute>30</minute>
       <second>0</second>
       <year>1999</year>
       <month>12</month>
       <day>30</day>
      </datetime>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::DaySection</enum>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::UTC</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="TaskButton">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>60</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(39, 39, 39);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Add task</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="icon">
     <iconset>
      <normaloff>../../Downloads/th (1).jpg</normaloff>../../Downloads/th (1).jpg</iconset>
    </attribute>
    <attribute name="title">
     <string/>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
